rendering/glb.ksy

```diff
@@ -0,0 +1,69 @@
+meta:
+  id: glb
+  title: GL Transmission Format, binary container
+  file-extension: glb
+  endian: le
+  license: MIT
+  xref:
+    mime: model/gltf-binary
+    wikidata: Q28135989
+
+doc: |
+  glTF is a format for distribution of 3D models optimized for being used in software
+
+doc-ref: https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#binary-gltf-layout
```

Comment:
Link to a specific commit rather than `master` which changes over time:

```suggestion
doc-ref: https://github.com/KhronosGroup/glTF/tree/2354846/specification/2.0#binary-gltf-layout
```